tests/misc/test_split.py

```diff
@@ -103,37 +103,39 @@
 áéíóú/áéíóú/áéíóú/
 """
 
+test_data_lines = test_data.strip().splitlines()
 
-class SplitTests(unittest.TestCase):
+
+class TestSplit:
 
     """Test split."""
 
-    def test_split(self):
+    @pytest.mark.parametrize('cmd, out',
+                             [case.split('/')[:-2]
+                              for case in test_data_lines])
```

Comment:
In fact I didn't try very hard to understand what the test was about initially... :blush: 

I have implemented a solution used `namedtuple` as suggested, and used a parametrized fixture to make sure all tests use the same set of parameters, please check it out and let me know what you guys think.